### PR TITLE
Image

### PR DESCRIPTION
**Pull request by Barskyi**
add:
1.CloudService
2.Schema Image
3. UserRout, AuthRout - correct some cods
4. Routes/Photo 5.Repo/users add unique()
6.Create Tags
7.Create Repo/Photo
8.Refactoring doc-string + add new column, delete count_photo (User) 9.main.py add (photo), add title, discription and orange worlds :alien:

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -6,16 +6,18 @@
...
+from src.routes import auth, users, photo
 
-app = FastAPI(title="ImageHUB")
+app = FastAPI(title="ImageHUB", description="Welcome to ImageHUB API",
+              swagger_ui_parameters={"syntaxHighlight.theme": "obsidian"})
 
 BASE_DIR = Path(__file__).parent
 directory = BASE_DIR.joinpath("src").joinpath("static")
-app.mount("/static", StaticFiles(directory=BASE_DIR/"src"/"static"), name="static")
+app.mount("/static", StaticFiles(directory=BASE_DIR / "src" / "static"), name="static")
```
Comment (Kostiantyn78) [inline main.py]: directory=BASE_DIR / "src" / "static" необходимо заменить на directory=directory

**Review by Kostiantyn78 — CHANGES_REQUESTED** [review]
Need changes

File: main.py (hunk 1 above)
Comment (Barskyi) [reply main.py]: i did it

**Review by Kostiantyn78 — APPROVED** [review]
Ok

**Pull request merged**